### DISCUSSION TITLE
Publishing my project on the core repo

### DISCUSSION THREAD

**Discussion by islemci** [Homebrew/discussions#5792] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.4.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: 8c868ba70e526f1f00561470d58fcbaddb1b4e48
Last commit: 2 days ago
Branch: stable
Core tap JSON: 07 Dec 16:25 UTC
Core cask tap JSON: 07 Dec 16:25 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.6 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.6/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 16.0.0 build 1600
Git: 2.46.2 => /opt/homebrew/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 15.0.1-arm64
CLT: 16.0.0.0.1.1724870825
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hello everyone,
I am sorry if this question was answered somewhere else, but I failed in understanding how I can even package my application.

I would like to add some resources that might be helpful while answering:
Github Repo for the project I would like to publish in the core repo:
https://github.com/islemci/cliwrap
Formulae is hosted here:
https://github.com/islemci/homebrew-cliwrap

I've deployed my own tap via `brew tap islemci/cliwrap` and `brew install cliwrap`. Now I want to add the formulae to the main repo, but I didn't want to act before not knowing how to commit to it, or should i release it to the main repo in the first place.

If someone experienced about this could help me, it would be really nice! Thanks in advance.

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew should help you

**Reply by islemci** [reply]
So over [here](https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff), there is a requirement of "being known". The project is new (~2-3 days in development), but the latest release has been tested with MacOS and Linux-based systems via Homebrew, and even tested with FreeBSD. It is a simple app, from what it is trying to achieve and how it works, and I am not sure if we even need to wait for ~75 stars.

We got some positive feedback from the community over at a [Turkish forum](https://btt.community/t/cliwrap-terminal-kullanim-ozeti/15469/10), if that would be supporting our application too! :) (sorry, you would have to translate the contents)

^edit: fix link

**Reply by SMillerDev** [reply, marked as answer]
The project being younger than 30 days is a reason for rejection from Homebrew/core and the project not being popular enough in terms of stars is indeed another. If you're submitting your own software that wouldn't be cause for rejection, but it's not a great look for the popularity.

This seems like a nice software, but you're right that it does not seem not notable enough for us to package in Homebrew/core for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere.

**Reply by islemci** [reply]
Thank you! Seems like we will wait for some time.
